src/config.js

```diff
@@ -50,7 +50,16 @@ const configuration = {
      * @returns {boolean}
      */
     isProduction () {
-        return this.env === this.PRODUCTION;
+        return this ? this.env === this.PRODUCTION : false;
```

Comment:
How does it happen that `this` does not exist? And if it does not exist it probably means the method is not used properly.

Reply:
done